### ISSUE TITLE
Regression: Component can't be left empty

### ISSUE THREAD

**Issue by 123dev** [magicdude4eva/graylog-jira-alarmcallback#8]
Hi Gerd,

Regression in v1.0.5
When component field is left blank

```
2016-04-20_14:59:06.72837 INFO  [JiraAlarmCallback] No existing open JIRA issue for MD5=2a1f99094b86a40e47c82099d5b9391c and filter-query='AND Status not in (Closed, Done)'
2016-04-20_14:59:07.34254 ERROR [JiraAlarmCallback] Error creating JIRA issue=Failed to create issue, Cause=400 Bad Request: {"errorMessages":[],"errors":{"components":"Component name '' is not valid"}}
2016-04-20_14:59:07.34385 WARN  [AlertScannerThread] Alarm callback <Graylog JIRA integration plugin> failed. Skipping.
2016-04-20_14:59:07.34736 org.graylog2.plugin.alarms.callbacks.AlarmCallbackException: Failed creating new issue
2016-04-20_14:59:07.34738       at com.bidorbuy.graylog.alarmcallbacks.jira.JiraIssueClient.createJIRAIssue(JiraIssueClient.java:186)
2016-04-20_14:59:07.34738       at com.bidorbuy.graylog.alarmcallbacks.jira.JiraIssueClient.trigger(JiraIssueClient.java:75)
2016-04-20_14:59:07.34738       at com.bidorbuy.graylog.alarmcallbacks.jira.JiraAlarmCallback.call(JiraAlarmCallback.java:150)
2016-04-20_14:59:07.34739       at com.bidorbuy.graylog.alarmcallbacks.jira.JiraAlarmCallback.call(JiraAlarmCallback.java:126)
2016-04-20_14:59:07.34739       at org.graylog2.periodical.AlertScannerThread.doRun(AlertScannerThread.java:113)
2016-04-20_14:59:07.34739       at org.graylog2.plugin.periodical.Periodical.run(Periodical.java:83)
2016-04-20_14:59:07.34741       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
2016-04-20_14:59:07.34748       at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
2016-04-20_14:59:07.34778       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
2016-04-20_14:59:07.34808       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
2016-04-20_14:59:07.34837       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
2016-04-20_14:59:07.34870       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
2016-04-20_14:59:07.34910       at java.lang.Thread.run(Thread.java:745)
2016-04-20_14:59:07.34945 Caused by: net.rcarz.jiraclient.JiraException: Failed to create issue
2016-04-20_14:59:07.34974       at net.rcarz.jiraclient.Issue$FluentCreate.executeCreate(Issue.java:116)
2016-04-20_14:59:07.35003       at net.rcarz.jiraclient.Issue$FluentCreate.execute(Issue.java:59)
2016-04-20_14:59:07.35117       at com.bidorbuy.graylog.alarmcallbacks.jira.JiraIssueClient.createJIRAIssue(JiraIssueClient.java:181)
2016-04-20_14:59:07.35242       ... 12 more
2016-04-20_14:59:07.35286 Caused by: net.rcarz.jiraclient.RestException: 400 Bad Request: {"errorMessages":[],"errors":{"components":"Component name '' is not valid"}}
2016-04-20_14:59:07.35363       at net.rcarz.jiraclient.RestClient.request(RestClient.java:160)
2016-04-20_14:59:07.35400       at net.rcarz.jiraclient.RestClient.request(RestClient.java:182)
2016-04-20_14:59:07.35451       at net.rcarz.jiraclient.RestClient.request(RestClient.java:200)
2016-04-20_14:59:07.35491       at net.rcarz.jiraclient.RestClient.post(RestClient.java:290)
2016-04-20_14:59:07.35530       at net.rcarz.jiraclient.RestClient.post(RestClient.java:334)
2016-04-20_14:59:07.35567       at net.rcarz.jiraclient.Issue$FluentCreate.executeCreate(Issue.java:114)
2016-04-20_14:59:07.35648       ... 14 more
```

This was working ok in version 1.0.4

Thanks

**Comment by magicdude4eva** [comment]
Fixed in release [1.0.6](https://github.com/magicdude4eva/graylog-jira-alarmcallback/releases/tag/1.0.6)

**Comment by 123dev** [comment]
Thanks Gerd.

I can confirm that this is indeed working in 1.0.6 :)
